module/applications/activity/activity-usage-dialog.mjs

```diff
@@ -170,7 +170,6 @@ export default class ActivityUsageDialog extends Dialog5e {
       case "concentration": return this._prepareConcentrationContext(context, options);
       case "consumption": return this._prepareConsumptionContext(context, options);
       case "creation": return this._prepareCreationContext(context, options);
-      case "footer": return this._prepareFooterContext(context, options);
```

Comment:
Seems this was already being called in `super`, so no need to call it again here.